keyboards/miiiw/common/gw6606.c

```diff
@@ -0,0 +1,201 @@
+#include "quantum.h"
```

Comment:
Could you add a GPL2+ compatible license header here?  All of the source files need to have a license explicitly listed.

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Reply:
Fixed, have added GPLv2 license as the other source files.

Reply:
Changed according to suggestion.